tremor-script/src/ast/aggregate_fn.rs

```diff
@@ -0,0 +1,107 @@
+use super::{BaseExpr, NodeMetas};
+use crate::ast::raw::{ExprsRaw, IdentRaw};
+use crate::ast::upable::Upable;
+use crate::ast::{Exprs, Helper, Ident};
+use crate::lexer::Location;
+use serde::Serialize;
+
+#[derive(Clone, Debug, PartialEq, Serialize)]
+pub struct RawAggregateFnDecl<'input> {
+    /// public because lalrpop
+    pub start: Location,
+    /// public because lalrpop
+    pub end: Location,
+    /// public because lalrpop
+    pub name: IdentRaw<'input>,
+    /// public because lalrpop
+    pub body: RawAggregateFnBody<'input>,
+}
+
+#[derive(Clone, Debug, PartialEq, Serialize)]
+pub struct RawAggregateFnBody<'input> {
```

Comment:
just for the sake of consistent naming 
```suggestion
pub struct AggregateFnBodyRaw<'input> {
```